corehq/apps/geospatial/templates/geospatial/partials/delete_saved_area_modal.html

```diff
@@ -0,0 +1,30 @@
+{% load i18n %}
+
+<div class="modal fade" id="delete-saved-area-modal" tabindex="-1" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <h4 class="modal-title">
+          Are you sure you want to delete this saved area?
```

Comment:
translation?

Reply:
resolved in 02da8893ae2bed12b955dadc0c477feafe63d105